api/src/templates/certificate/index.js

```diff
@@ -86,6 +88,11 @@ const phase1 = async (young, batch = false) => {
   const departureDate = await getDepartureDateSession(meetingPoint, session, young, cohort);
   const returnDate = await getReturnDateSession(meetingPoint, session, young, cohort);
 
+  if (!cohesionCenter) {
+    capture(ERRORS.NO_COHESION_CENTER_FOUND);
```

Comment:
Au regard de la fonction `capture`, je pense que cela te fera une remontée de type "Error not defined well"
pourquoi ne pas laisser le catch de la route récupérer le throw ?